configman/def_sources/__init__.py

```diff
@@ -14,13 +14,21 @@
 definition_dispatch = {
   collections.Mapping: for_mappings.setup_definitions,
   type(for_modules): for_modules.setup_definitions,
-  #list: for_list.setup_definitions,
   str: for_json.setup_definitions,
   unicode: for_json.setup_definitions,
-  #type: for_class.setup_definitions,
 }
 
 
+try:
+    from configman.def_sources import for_argparse
+    import argparse
```

Comment:
This line isn't needed because, I think, `for_argparse` does an `import argparse` in it, doesn't it?